src/composables/types/diagnosis.ts

```diff
@@ -0,0 +1,82 @@
+import { 
+  AcutenessType,
+  AgeRequestType,
+  ExtrasType,
+  EvidenceType,
+  SexType,
+  PrevalenceType,
+  SeverityType,
+  TriageLevelType,
+} from '@/composables/types';
+
+export type ConditionDetailsModelCategoryType = {
+  id?: string,
+  name?: string,
+}
+
+export type ConditionDetailsModelType = {
+  icd10_code?: string,
+  category?: ConditionDetailsModelCategoryType,
+  prevalence?: PrevalenceType,
+  severity?: SeverityType,
+  acuteness?: AcutenessType,
+  triage_level?: TriageLevelType,
+  hint?: string,
+  has_patient_education?: boolean,
+}
+
+export type ConditionProbabilityType = {
+  id: string,
+  name: string,
+  common_name?: string,
+  probability: number,
+  condition_details?: ConditionDetailsModelType,
+}
+
+export enum ChoiceId {
+  Absent = 'absent',
+  Present = 'present',
+  Unknown = 'unknown',
+}
```

Comment:
Enums are considered harmful, there are better ways to structure data than enums ;) I've posted it to the #dev-frontend channel some time ago :) 
https://www.youtube.com/watch?v=jjMbPt_H3RQ

Reply:
I will check and replace